src/functional/fencil_processors/builders/cpp/bindings.py

```diff
@@ -171,22 +171,23 @@ def make_argument(
 
 
 def create_bindings(
-    target: source_modules.Function, target_header: str
+    source_module: source_modules.SourceModule, language: CppLanguage = CPP_DEFAULT
```

Comment:
While it is true that this particular bindings creator does not require the full `source_module`, when caching and name mangling for versioning purposes come into play that might not be true anymore. In addition, we want to call each step directly with the output of the previous step.

This means, it might be better to combind this `language` with the `SourceModule.language` tag to further streamline the interface.